packages/story-editor/src/types/storyTriggers.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import type { State } from './storyProvider';
+
+export type StoryTriggersState = (
+  | ((eventType: string, listener: any) => () => void)
```

Comment:
Oh! I actually set this to `any` to leave it for later, assuming it would be complained about. Forgot completely and eslint didn't complain for some reason. 🤔